src/api/app/views/webui/comment/beta/_comments_thread.html.haml

```diff
@@ -1,3 +1 @@
-.timeline-item
```

Comment:
This would render an extra `timeline-item` div which made the HTML a bit harder to follow since there were timeline items rendered in other timeline items.

We have to keep using a partial here since this is called only in the `webui/comments_controller` with AJAX. See:
- https://github.com/openSUSE/open-build-service/blob/5e8fef2198cd0c1665d04bab6e9de3e8f401740b/src/api/app/controllers/webui/comments_controller.rb#L19
- https://github.com/openSUSE/open-build-service/blob/5e8fef2198cd0c1665d04bab6e9de3e8f401740b/src/api/app/controllers/webui/comments_controller.rb#L58
- https://github.com/openSUSE/open-build-service/blob/5e8fef2198cd0c1665d04bab6e9de3e8f401740b/src/api/app/controllers/webui/comments_controller.rb#L84

The AJAX calls are handled here:
- https://github.com/openSUSE/open-build-service/blob/5e8fef2198cd0c1665d04bab6e9de3e8f401740b/src/api/app/assets/javascripts/webui/comment.js#L43
- https://github.com/openSUSE/open-build-service/blob/5e8fef2198cd0c1665d04bab6e9de3e8f401740b/src/api/app/assets/javascripts/webui/comment.js#L61
- https://github.com/openSUSE/open-build-service/blob/5e8fef2198cd0c1665d04bab6e9de3e8f401740b/src/api/app/assets/javascripts/webui/comment.js#L81